src/lucky/tags/base_tags.cr

```diff
@@ -125,6 +125,51 @@ module Lucky::BaseTags
       end
     end
 
+    def {{method_name.id}}(
+        content : Nil,
+        options = EMPTY_HTML_ATTRS,
+        attrs : Array(Symbol) = [] of Symbol,
+        **other_options
+      ) : Nil
+      \{%
+        raise <<-ERROR
+          HTML tags content must be a String or Lucky::AllowedInTags object, not nil.
+          Be sure the content isn't a nil value. For nested HTML tags, use the block
+          method overload
+
+          Example...
+
+            {{method_name.id}}(title, class: "header")
+
+            to
+
+            {{method_name.id}}(class: "header") do
+              title "My Site Title"
+            end
```

Comment:
Idk how I feel about this. I've accidentally passed in a nil a couple times but it was always a nilable value. Could that be the example instead and we just add a sentence to the above that's like `Maybe you accidentally called a method that renders html?` but more eloquently of course

Reply:
Yeah, this felt a little weird to me too, but I wanted to showcase the original error since it felt like a fairly common use case. Maybe I do something like this?

```crystal
<<-ERROR
HTML tags content must be a String or Lucky::AllowedInTags object, not nil.

Maybe you called a method that also renders html? (e.g. title, address, summary, etc...)

Try this...

  if value = some_nilable_value
    {{method_name.id}}(value, class: "header")
  end

ERROR
```

Reply:
Still feels like that sentence and the example are connected when they're not. I think the error without that sentence wuold be enough for the user to have something to go on to figure out where they went wrong.

Reply:
So remove the example code completely and just leave the error? Or you're saying remove the "Maybe you called a method" line?

Reply:
The last thing about accidentally calling a method that renders html is what I think can go